docs/markdown/releasenotes.md

```diff
@@ -5,8 +5,10 @@
 ### New features
 
 * Verified support for Java 16 and 17.
-* Added support for vendoring Go Mod dependencies using [detect.go.mod.dependency.types.excluded=VENDORED](properties/detectors/go.md#go-mod-dependency-types-excluded) to exclude *test* and *build system* dependencies from Go modules
-  declaring a version prior to `Go 1.16`.
+* Added support for vendoring Go Mod dependencies using [detect.go.mod.dependency.types.excluded=VENDORED](properties/detectors/go.md#go-mod-dependency-types-excluded) to exclude *
+  test* and *build system* dependencies from Go modules declaring a version prior to `Go 1.16`.
+* Added a feature that allows users to configure [solution_name] to fail when policies of a certain name are vialated.
```

Comment:
@asallette Line 10+11 are the new release note, all other changes are as a result to applying a new formatter.